src/pages/404/404.module.sass

```diff
@@ -0,0 +1,9 @@
+.container
```

Comment:
`container` class is useless because it already defaulted to `width: 100%` because `div` is defaulted to `display: block`, so it takes all available horizontal space.